src/syscalls/deprecated_syscall_handler.rs

```diff
@@ -335,7 +377,11 @@ mod tests {
 
         let ids_data = ids_data!["syscall_ptr"];
 
-        let hint_data = HintProcessorData::new_default(GET_BLOCK_TIMESTAMP.to_string(), ids_data);
```

Comment:
Why don't we keep `hint_code.rs` instead of manually replacing the string everywhere?